### PR TITLE
Welcome to our new storage maintainers

### PR DESCRIPTION
**Pull request by mestery**
Signed-off-by: Kyle Mestery <mestery@mestery.com>

### CODE REVIEW

**Comment by mestery** [comment]
@ artek-koltun and @intelfisz please review

**Review by artek-koltun — APPROVED** [review]

**Review by intelfisz — COMMENTED** [review]
Syntactically everything is fine. I am not going to approve myself as a maintainer though, so waiting for other folks to approve :-)

**Review by dandaly — APPROVED** [review]

**Pull request merged**
